README.md

```diff
@@ -473,6 +473,20 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 4.9 Refactor regularly using static analysis tools
```

Comment:
Hi @TheHollidayInn, a couple observations regarding the formatting.

From what I can observe, it appears as if the formatting for the rest of _this_ readme has the [⬆ Return to top](https://github.com/i0natan/nodebestpractices/pull/176/files#diff-04c6e90faac2675aa89e2176d2eec7d8R474) link at the bottom of each section.

I also observed that there are usually 2 `<br/>` tags between sub-sections but 3 `<br/>` tags before the `⬆ Return to top` link. Is this the correct formatting? Perhaps someone with more context could clarify if this is the desired format.

Please could you move this section above the `⬆ Return to top` link, to maintain the consistency within the document. 

Thanks